src/wp-admin/includes/revision.php

```diff
@@ -66,7 +66,18 @@ function wp_get_revision_ui_diff( $post, $compare_from, $compare_to ) {
 
 	$return = array();
 
-	foreach ( _wp_post_revision_fields( $post ) as $field => $name ) {
+	$revision_screen_fields = _wp_post_revision_fields( $post );
+
+	/**
+	 * Filters the fields to display on the revisions screen.
+	 *
+	 * Developers can use this filter to control the display of fields on the revisions screen.
+	 *
+	 * @since 6.4.0
+	 */
+	$revision_screen_fields = apply_filters( 'wp_revision_ui_fields', $revision_screen_fields, $post );
```

Comment:
What's the reason for introducing a new hook an not using `_wp_post_revision_fields`?

Reply:
I have the same question ^.

Reply:
added param documentation

Reply:
@adamsilverstein The since is the wrong place.

Reply:
fixed since placement